assets/javascripts/discourse-assign/initializers/extend-for-assigns.js.es6

```diff
@@ -119,6 +121,19 @@ function initialize(api) {
     before: "top",
   });
 
+  api.addAdvancedSearchOptions({
+    inOptionsForUsers: [
+      {
+        name: I18n.t("search.advanced.in.assigned"),
+        value: "assigned",
+      },
+      {
+        name: I18n.t("search.advanced.in.unassigned"),
+        value: "unassigned",
+      },
+    ],
+  });
+
```

Comment:
Do we need to check `currentUser.can_assign` here? We don't want to show this to all users